p2p/providers/peerstore_provider/persistence/provider.go

```diff
@@ -28,6 +28,7 @@ func (*persistencePeerstoreProvider) Create(bus modules.Bus, options ...persiste
 	persistencePSP := &persistencePeerstoreProvider{
 		IntegratableModule: *base_modules.NewIntegratableModule(bus),
 	}
+	persistencePSP.SetBus(bus)
```

Comment:
This should probably be:
```suggestion
	bus.RegisterModule(persistencePSP)
```